UPGRADING

```diff
@@ -78,7 +78,9 @@ PHP 8.3 UPGRADE NOTES
 
 - Intl:
   . datefmt_set_timezone (and its alias IntlDateformatter::setTimeZone) 
-    now returns true on sucess, previously null was returned.
+    now returns true on success, previously null was returned.
+  . breakiterator::setText now returns false on failure, previously
```

Comment:
```suggestion
  . IntlBreakIterator::setText() now returns false on failure, previously
```

Function brances (`()`) in the upgrading file are sometimes omitted, sometimes not, but personally, I'd prefer to have them if possible.